.github/workflows/ci.yml

```
@@ -173,6 +173,13 @@ jobs:
         # Prefix the list here with "+" to use these queries and those in the config file.
         # queries: ./path/to/local/query, your-org/your-repo/queries@main
 
+    - name: Build and install ECM
+      run: |
+        cd ..
+        git clone https://invent.kde.org/frameworks/extra-cmake-modules.git
+        cmake -S extra-cmake-modules -B extra-cmake-modules/build $CMAKE_ARGS
+        cmake --build extra-cmake-modules/build --target install
+
```

Can we also have some basic test that the installed QML integration files are there and have something reasonable inside? Like grep a line in `qmldir` and `*qmltypes`... Just to make sure it doesn't silently fall apart, because right now QML integration is not checked at all.

---

done